lib/tasks/BackbeatTask.js

```diff
@@ -1,37 +1,30 @@
 const BackOff = require('backo');
-const joi = require('joi');
-
-const BACKOFF_PARAMS = { min: 1000, max: 300000, jitter: 0.1, factor: 1.5 };
-
-const backbeatTaskConfigJoi = {
-    retryTimeoutS: joi.number().default(300),
-};
 
 /**
  * @class BackbeatTask
  */
 class BackbeatTask {
-    /**
-     * @constructor
-     * @param {Object} [config] - config object containing various
-     *   task-specific configuration
-     * @param {Number} [config.retryTimeoutS] - timeout of retries, in
-     *   seconds
-     */
-    constructor(config) {
-        this.config = joi.attempt(config || {}, backbeatTaskConfigJoi,
-                                  'invalid backbeat task config');
+
+    constructor() {
+        this.retryParams = {
+            timeoutS: 300,
+            backoff: {
+                min: 1000,
+                max: 300000,
```

Comment:
~300000~ 30000

Reply:
It's 300K ms. as in 300S which is the timeout. Isn't that correct?

Reply:
I see, it looks like the other values set at `30000` need to be updated in that case.

Reply:
I just saw that, thanks for pointing out. I updated the values correctly to be 300K